crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1333,7 +1340,86 @@ pub struct StripeWebhookObjectResource {
 #[derive(Debug, Deserialize)]
 pub struct StripeWebhookObjectEventType {
     #[serde(rename = "type")]
-    pub event_type: String,
+    pub event_type: WebhookEvenType,
+    pub data: serde_json::Value,
+}
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct DataObject {
+    pub object: ObjectData,
+}
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct ObjectData {
+    pub id: String,
+    pub amount: i32,
+    pub currency: String,
+    pub reason: String,
+    pub created: i64,
+    pub evidence_details: EvidenceDetails,
+    pub status: DisputeStatus,
```

Comment:
the values that are not there always, make those as option, else deserialization will fail